Makefile

```diff
@@ -15,3 +15,5 @@ coverage:
 clean:
 	hatch run clean
 
+update-abi:
+	./scripts/update_abi
```

Comment:
nitpick: it is expected/consistent to have newline at end file, please review $EDITOR settings